src/Rainflow.jl

```diff
@@ -71,23 +69,27 @@ mutable struct Cycles_bounds #
     min_mean::Float64
     max_mean::Float64
     max_range::Float64
+    min_R::Float64
+    max_R::Float64
 end
 
-show(io::IO,x::Cycles_bounds) = print(io, "Cycles_bounds : min mean value=", x.min_mean, ", max mean value=", x.max_mean, ", max range=",x.max_range)
+show(io::IO,x::Cycles_bounds) = print(io, "Cycles_bounds : min mean value=", x.min_mean, ", max mean value=", x.max_mean, ", max range=",x.max_range, ", min R=", x.min_R, ", max R=",x.max_R)
 
-""" Find the minimum and maximum mean value and maximum range from a vector of cycles. """
+# Find the minimum and maximum mean value and maximum range from a vector of cycles
 function find_boundary_vals(cycles::Array{Cycle,1})
-    bounds = Cycles_bounds(Inf, -Inf, -Inf)
+    bounds = Cycles_bounds(Inf, -Inf, -Inf, Inf, -Inf)
     for cycle in cycles
         cycle.mean > bounds.max_mean && setfield!(bounds, :max_mean, cycle.mean)
         cycle.mean < bounds.min_mean && setfield!(bounds, :min_mean, cycle.mean)
         cycle.range > bounds.max_range && setfield!(bounds, :max_range, cycle.range)
+        cycle.Rvalue < bounds.min_R && setfield!(bounds, :min_R, cycle.Rvalue)
+        cycle.Rvalue > bounds.max_R && setfield!(bounds, :max_R, cycle.Rvalue)
     end
     return bounds
 end
 
-""" Returns the range index the value is belonging in """
-function find_range{T<:Real}(interval::Array{T,1},value)
+# Returns the range index the value is belonging in
+function find_range(interval,value)
```

Comment:
Please keep the function as shown below, as the function assumes the interval is a vector
`function find_range(interval::Array{T,1},value) where {T <: Real}`